tests/system/apmserver.py

```diff
@@ -212,7 +215,7 @@ def check_for_no_smap(self, doc):
             assert "sourcemap" not in frame
 
 
-class ClientSideBaseTest(ServerBaseTest):
+class ClientSideBaseTest(ElasticTest):
```

Comment:
With this change, also the [ClientSideTest](https://github.com/jalvz/apm-server/blob/905aef392efc1a6cd07f86ac8bcb5a6fe3cd4a86/tests/system/test_requests.py#L136) and the [RateLimitTest](https://github.com/jalvz/apm-server/blob/905aef392efc1a6cd07f86ac8bcb5a6fe3cd4a86/tests/system/test_requests.py#L214) will be derived from `ElasticTest` which includes the index creation and template loading to ES and slows down tests.

Reply:
yeah, but multiple inheritance doesn't play well here with the way in which we set configuration attributes because if they conflict you overwrite the configuration from one base class with the other one (i actually ran into that). so i think is better to have always 1 base class for simplicity, even if it is slower (it takes ages anyway).

pushed e8a93f982cb21468234142d6bb959aa5bec99cf7 (and 0d14abc258a444419998566e679d92fb6f113c3d)

Reply:
ok